cql3/expr/prepare_expr.cc

```diff
@@ -1260,10 +1260,9 @@ binary_operator prepare_binary_operator(binary_operator binop, data_dictionary::
 
         if (!rhs_is_null) {
             expression binop_expr = binop;
```

Comment:
This line could be removed as well. `binop_expr` was created so that we could have `expression&` to give to the printer. It wouldn't work with a reference to `binary_operator`.

Reply:
removed in v2.